### ISSUE TITLE
Too wordy

### ISSUE THREAD

**Issue by bcaffo** [jhpce-jhu/jhpce_mkdocs#6]
Honestly, the docs are way too wordy. I think we should thin them out substantially.

**Comment by jeffreytunison** [comment]
Agreed. Too many pages on the old site are too dense. But that's why I'm driven to revise our new site rather than just removing SGE information from existing pages.

- Having tables of contents in every doc makes it a lot easier to create pages that are longer yet still useful. 
- We should as much as possible add anchor tags so we can make it as easy as possible for users to jump to information rather than having to read through dense text to find their desired info.
